pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     "jsonschema>=4.6,<5.0",
     "libcomps>=0.1.20.post1,<0.2",
     "productmd~=1.33.0",
-    "pulpcore>=3.49.11,<3.70",
+    "pulpcore>=3.49.11,<3.85",
```

Comment:
We should add lowerbounds to 3.70 because CONTENT_ORIGIN changes are incompatible with rpm pre-3.70 support.

Reply:
Actually I don't think that this holds true.
What this PR does is add extra validation for the case where CONTENT_ORIGIN is not set, what was impossible prior to 3.70, so it is not really incompatible. Or is there something else that would actually break?

Reply:
Aye - we intentionally want this to work regardless of what the user does. In this context, the only thing core/3.70 does is *allow for* CONTENT_ORIGIN to be null - prior, pulp won't even start in that state.